src/EditorFeatures/Test/Preview/PreviewWorkspaceTests.cs

```diff
@@ -238,6 +238,19 @@ public void TestPreviewWorkspaceDoesNotLeakSolution()
             solutionObjectReference.AssertReleased();
         }
 
+        [Fact]
+        public void TestPreviewWorkspaceDoesNotLeakItself()
```

Comment:
Oh nice adding tests for workspaces generally and also PreviewWorkspace specifically.